documentation/usage/gradle/index.md

```diff
@@ -176,6 +176,32 @@ flyway {
 
 For details on how to setup and use custom Gradle configurations, see the [official Gradle documentation](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.ConfigurationContainer.html).
 
+### Using Flyway Database Types
+
+For some Flyway database types (non core), like [Cloud Spanner](/documentation/database/cloud-spanner) or [SQL Server](/documentation/database/sqlserver), you'll need to actually use a `buildScript` to get your
+Gradle commands to work properly. You need to enable putting the database type on the build classpath, and not
+the project classpath.
+
+Something like the following should have you set:
+
+```groovy
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath "org.flywaydb:flyway-mysql:${flywayVersion}"
```

Comment:
Can you replace `${flywayVersion}` with `{{ site.flywayVersion }}` which is the GitHub pages way of using our variable for the site version